packages/celotool/src/lib/genesis_constants.ts

```diff
@@ -40,3 +40,74 @@ export const CONTRACT_OWNER_STORAGE_LOCATION =
   '0xb53127684a568b3173ae13b9f8a6016e243e63b6e8ee1178d6a717850b5d6103'
 export const ISTANBUL_MIX_HASH =
   '0x63746963616c2062797a616e74696e65206661756c7420746f6c6572616e6365'
+
+export const INITIAL_ACCOUNTS = [
+  {
+    address: '0x36940810BfDB329B31e38d3e97aFD673081B497C', // ReserveCo genesis address
+    balance: '120000000000000000000000000', // 120MM
+  },
+  {
+    address: '0xfCf982bb4015852e706100B14E21f947a5Bb718E', // LLC genesis address
+    balance: '200000000000000000000000000', // 200MM
```

Comment:
why aren't these split into several addresses?

Reply:
Had this same question - the answer I was given from Asa was that the split addresses are for amounts that are planned to be used often and so are vulnerable. 
The reserve address here will immediately be transferred to the reserve contract so its fine, and the LLC eventually gets transferred to the LLC treasury

Reply:
I see -- thanks for the explanation  👍